### ISSUE TITLE
Left room navigation is really hard to click on Mobile

### ISSUE THREAD

**Issue by MadLittleMods** [gitterHQ/gitter#806]
*As reported by @sourcefrog in the [gitterHQ/gitter room](https://gitter.im/gitterHQ/gitter?at=55c14b7b2c1b3bec3198a9c7).*

On a small mobile phone, the navigation is really hard to click. Normal taps just seem to close the menu without navigating to the room. I am testing this on a Moto E, Android 5.1 in Chrome.

@chamila-c brought up a workaround solution to long tap an item which I can also get working.

> "[It's the] linger of the finger"

This is probably caused by the lack of semantic, focusable elements for the room list items; such as a `<a>` anchor, `<button>`, or `tabindex` attribute. Semantic markup has been brought up in #538.

![](http://i.imgur.com/R3ZwOPc.png)

---

Keep in mind that I have no problem navigating the room menu on a Nexus 7 tablet but this does use a different breakpoint without the hamburger menu.

**Comment by chamila-c** [comment]
:+1:

**Comment by sourcefrog** [comment]
thanks,  :+1:

**Comment by AverageMarcus** [comment]
:+1: 

The long press doesn't seem to always work for me.

**Comment by 304NotModified** [comment]
+1

Long press works around 5% of the times.  I'm now changing the url als work around - but not happy with it. 

This seems obvious to fix?

**Comment by trevorah** [comment]
Looks like we had a clash between hammerjs and fastclick. I've got a fix thats going through our internal testing now.

**Comment by 304NotModified** [comment]
Good news!

**Comment by k-bx** [comment]
Is this live already? Rooms switching still doesn't work! Thanks.

**Comment by trevorah** [comment]
It should be live now, it went out in a deploy that happened about an hour ago. Is it still broken for you?

**Comment by k-bx** [comment]
Absolutely. Even a long-press hack stopped working :/

I'm on SGS3 Chrome.

**Comment by AverageMarcus** [comment]
It's working fine for me now.

**Comment by k-bx** [comment]
Sorry! Just did additional "refresh", re-checked -- and it seems working now.

**Comment by trevorah** [comment]
@k-bx awesome, thanks for re-checking!

**Comment by k-bx** [comment]
Thanks for implementation :)
